plugin/js/parsers/BakaTsukiImageCollector.js

```diff
@@ -111,7 +112,11 @@ ImageElementConverter.prototype.replaceWithImagePageUrl = function (images) {
     let imageInfo = images.get(that.imagePageUrl);
     if (imageInfo != null && that.element.parentElement != null) {
         let newImage = imageInfo.createImageElement();
-        that.element.parentElement.replaceChild(newImage, that.element);
+        if(that.element.className === "thumb" && (imageInfo.isOutsideGallery || imageInfo.isCover)){
```

Comment:
imageInfo.isCover is never set to true or grabbed anywhere. This behavior should be changed in the future so that isCover is set to true for whatever image the checkmark is on for "Set Cover".